### PR TITLE
PWGLF: Add task for ID RAA

### PR DESCRIPTION
**Pull request by jpwgnr**
Task for the identified RAA measurement for the Summer Student Project.

### CODE REVIEW

File: Analysis/Tasks/PWGLF/CMakeLists.txt [hunk 1]
```diff
@@ -84,6 +84,11 @@ o2_add_dpl_workflow(raacharged
                     PUBLIC_LINK_LIBRARIES O2::Framework O2::AnalysisDataModel O2::AnalysisCore
                     COMPONENT_NAME Analysis)
 
+o2_add_dpl_workflow(raameasurement
```
Comment (jgrosseo) [inline Analysis/Tasks/PWGLF/CMakeLists.txt]: Let's not have measurement in the name. If this is identified RAA, maybe call it identified-raa

**Review by jgrosseo — APPROVED** [review]

**Review by jgrosseo — CHANGES_REQUESTED** [review]
Just change the file name

**Comment by jpwgnr** [comment]
Yes, you're right. I changed it. Thanks a lot for the spotting of that.

**Comment by jgrosseo** [comment]
Unfortunately, the copyright notice seems to be the old one (it was changed recently):
`/sw/SOURCES/O2/6704-slc8_x86-64/0//Analysis/Tasks/PWGLF/identifiedraa.cxx:1:1: error: missing or malformed copyright notice`
Please update from any file currently in the repository.

**Review by jgrosseo — DISMISSED** [review]

**Comment by jpwgnr** [comment]
Hi, I finally fixed everything. Would be nice if you could give the okay now. Thanks!

**Review by jgrosseo — APPROVED** [review]
Very good!

**Pull request merged**